src/CBLReplicator_Internal.hh

```diff
@@ -123,7 +123,19 @@ public:
         });
         _c4status = _c4repl->getStatus();
     }
-
+    
+    
+    static CBLReplicatorConfiguration defaultConfiguration()
+    {
+        CBLReplicatorConfiguration config = {};
+        config.replicatorType = kCBLReplicatorTypePushAndPull;
+        config.continuous = false;
+        config.maxRetries = -1;
+        config.maxRetryWaitTime = 0;
```

Comment:
`{}` is a shorthand for zeroing so setting 0 / false is redundant.

Reply:
Agreed. I have created https://issues.couchbase.com/browse/CBL-1906 to revise the name and meaning so until then, please accept the change.

Reply:
Once that has been changes, we can discard CBLDatabaseConfiguration_Default as well.